docs/src/use/core-concepts/glossary.md

```diff
@@ -0,0 +1,397 @@
+
+### Formatting (Rule)
+
+A rule that solely targets [formatting](#formatter-tool) concerns, such as semicolons and whitespace.
+These rules don't change application logic and are a subset of [Stylistic rules](#stylistic-rule).
+
+ESLint's built-in formatting rules are deprecated and will be removed in a later major version.
+ESLint recommends instead using a dedicated formatter such as [Prettier](https://prettier.io) or [dprint](https://dprint.dev).
+Alternately, the [ESLint Stylistic project](https://eslint.style) provides formatting-related lint rules.
+
+For more information, see [Deprecation of formatting rules](https://eslint.org/blog/2023/10/deprecating-formatting-rules).
+
+## G
+
+### Global
+
+A description of a global variable.
+
+Global variables inform lint rules that check for proper uses of [globals](#global).
+For example, the [`no-undef` rule](../../rules/no-undef) will create a violation for references to global variables not defined in the configured list of environments.
+
+[Flat configs](#flat-config) have globals defined as JavaScript objects.
+[Legacy configs](#legacy-config) have globals defined via predefined [environments](#environment).
+
+For information about configuring globals, see [Configure Language Options > Specifying Globals](../configure/language-options#specifying-globals).
+
+## I
+
+### Inline Config (Configuration Comment)
+
+A source code comment that configures a rule to a different severity and/or set of options.
+
+Inline configs use similar same syntax as [config files](#config-file-configuration-file) to specify any number of rules by name, their new severity, and optionally new options for the rules.
+For example, the following inline config comment simultaneously disables the `eqeqeq` rule and sets the `curly` rule to `"error"`:
+
+```js
+/* eslint eqeqeq: "off", curly: "error" */
+```
+
+For documentation on inline config comments, see [Rules > Using configuration comments](../configure/rules#using-configuration-comments).
+
+## L
+
+### Legacy Config
+
+The previous configuration file format for ESLint, now superseded by ["Flat" config](#flat-config).
+
+Legacy ELSint configurations are named in the format `.eslintrc.*` and allowed to be nested across files within sub-directories in a project.
+
+You can read more about the lifetime of legacy configurations in [ESLint's new config system, Part 1: Background](https://eslint.org/blog/2022/08/new-config-system-part-1).
+
+### Linter
+
+A [static analysis](#static-analysis) tool that can [report](#report) the results from running a set of [rules](#rule) on source code.
+Each rule may report any number of [violations](#violation) in the source code.
+
+ESLint is a commonly used linter for JavaScript and other web technologies.
+
+Note that a _linter_ is separate from [formatters](#formatter-tool) and [type checkers](#type-checker).
+
+### Logical (Rule)
+
+A [rule](#rule) that looks at logical issues in code.
```

Comment:
For clarity:

```suggestion
A [rule](#rule) that inspects how code functions to find problems.
```

Reply:
I like this change, though I'm concerned that _"functions"_ is an overloaded term. Maybe: _"operates"_?

Reply:
Ah, I like https://github.com/eslint/eslint/pull/18187/files#r1530870023 a lot. Will re-use that.